test/compeon/rack_tools/pipes/parse_oauth2_authorization_code_test.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+require 'compeon/rack_tools/pipes/parse_oauth2_authorization_code'
+
+module Compeon
+  module RackTools
+    module Pipes
+      class ParseOauth2AuthorizationCodeTest < Minitest::Test
+        AUTH_KEY = OpenSSL::PKey::RSA.new(512)
+
```

Comment:
```suggestion

        def setup
          Compeon::RackTools::Token.oauth2_public_key = AUTH_KEY.to_s
        end

        def teardown
          Compeon::RackTools::Token.oauth2_public_key = nil
        end

```

Reply:
https://github.com/COMPEON/compeon-rack_tools/pull/4/files#r283374335

Reply:
I'd rather leave it as it is instead of adding an additional setter for testing purposes only.

Reply:
We already has this discussion and we decided that we want to avoid stubs, we decided to add mock functionality to our mocks.